src/bindings/js/node/src/tensor.cpp

```diff
@@ -132,6 +132,39 @@ Napi::Value TensorWrap::get_data(const Napi::CallbackInfo& info) {
     }
 }
 
+std::map<napi_typedarray_type, ov::element::Type> corresponding_ov_type{
```

Comment:
Use unordered map instead (static const). Is this conversion required only here or maybe is used also in other places in JS bindings?
If yes util helper can be created.
Use `switch case` instead map is simpler as it will be one value into other value conversion no object with dynamic allocations.

Reply:
This map is equivalent to Python `const std::map<std::string, ov::element::Type>& dtype_to_ov_type();`  and `get_ov_type(py::dtype& dtype)` which are used multiple times.